### ISSUE TITLE
Is it possible to change the MySQL root password?

### ISSUE THREAD

**Issue by cnicodeme** [jide/clamp#21]
Hi,

By reading the source, I saw that the root password was used via `$this->getConfig('$.mysql.users.root.password')` so I believe it's possible.

I tried to modify my clamp.json file by adding the users.root.password like this : 
```
"users": {
            "root": {
                "password": "unbreakable_password"
            }
        }
```

Right after "options", in "mysql". But it doesn't seems to work.

Do I have to set it before running MySQL the first time? Or do I have to do the changes manually in MySQL, and this variable is to let Clamp know the root password?

Thanks in advance :)

**Comment by jide** [comment]
Hello,

Yes this should work if the password is set before mysql is setup, but not afterwards.

**Comment by cnicodeme** [comment]
I didn't succeed by creating the clamp.json file before, setting the root password, then setting the mysql server. I don't really know what happens (it couldn't bind the server to a local port).

But I did the following, and as far as I'm concerned, it works : 

1. I created the config file
2. Started clamp
3. Accessed MySQL and changed the root password + flush
4. Changed the clamp.json file to add the root password

And it works now :)
